examples/compiled/layer_line_co2_concentration.vg.json

```diff
@@ -97,7 +97,7 @@
             "update": {
               "stroke": {"scale": "color", "field": "decade"},
               "description": {
-                "signal": "\"decade: \" + (isValid(datum[\"decade\"]) ? datum[\"decade\"] : \"\"+datum[\"decade\"]) + \"; Year into Decade: \" + (format(datum[\"scaled_date\"], \"\")) + \"; CO2 concentration in ppm: \" + (format(datum[\"CO2\"], \"\"))"
+                "signal": "\"scaled_date: \" + (format(datum[\"scaled_date\"], \"\")) + \"; CO2 concentration in ppm: \" + (format(datum[\"CO2\"], \"\")) + \"; decade: \" + (isValid(datum[\"decade\"]) ? datum[\"decade\"] : \"\"+datum[\"decade\"])"
```

Comment:
It used to say "Year into Decade". Is this description still there somewhere?

Reply:
In the axis title, but I can't just make it field title because the "min" scaled_date. Let me hide aria for the text labels, then we can bring this back.

Reply:
ok